src/main/java/org/broadinstitute/hellbender/engine/ReferenceDataSource.java

```diff
@@ -28,6 +27,21 @@ public static ReferenceDataSource of(final Path fastaPath) {
         return new ReferenceFileSource(fastaPath);
     }
 
+    /**
+     * Initialize this data source using a fasta file.
+     *
+     * The provided fasta file must have companion .fai and .dict files.
+     *
+     * If {@code preserveFileBases} is {@code true}, will NOT convert IUPAC bases in the file to `N` and will NOT capitalize lower-case bases.
```

Comment:
Javadoc doesn't match code, this parameter doesn't appear in the method signature.

Also note that javadoc doesn't preserve newlines in its output. If you want newlines between text in Javadoc, put a `<p>` tag on each empty line.